corehq/motech/repeaters/models.py

```diff
@@ -1277,16 +1270,6 @@ def _get_retry_interval(last_checked, now):
     return interval
 
 
-def attempt_forward_now(repeater: Repeater):  # unused
-    from corehq.motech.repeaters.tasks import process_repeater
-
-    if not domain_can_forward(repeater.domain):
-        return
-    if not repeater.is_ready:  # only place that uses Repeater.is_ready
-        return
-    process_repeater.delay(repeater.id.hex)
-
-
```

Comment:
It looks like the next two methods below (`get_payload` and `send_request`) are also unused remnants of that earlier effort. Is there a reason they were not removed?

Reply:
No good reason. They were also part of the effort to simplify the labyrinthine workflow for sending payloads. But we know where to find them. I'll add an item to my to-do list to drop them too.